mujoco_warp/_src/constraint.py

```diff
@@ -139,6 +142,58 @@ def _efc_limit_slide_hinge(
     )
 
 
+@wp.kernel
+def _efc_limit_ball(
+  m: types.Model,
+  d: types.Data,
+  refsafe: bool,
```

Comment:
it kind of looks like you could just create `refsafe` directly inside of `_update_efc_row` and then you don't need to pass it around.

could you even do `refsafe = wp.static(m.opt.disableflags & ...` so that it's zero cost at runtime?

Reply:
the status of refsafe is now checked in `_update_efc_row`.

`wp.static` does not seem to work here so a todo is added for now.